source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -68,6 +70,27 @@ Even if documents move between source shards during replication,
 guarantee on the destination cluster. For more information, see 
 :ref:`c2c-sharded-clusters`.
```

Comment:
I mentioned in the description of DOCSP-45117 that the balancer / chunk migration stuff no longer applies, so L62- L71 isn't relevant anymore. We can instead 'summarize' the two sub-sections after it by replacing L62 - L71 with:

> {+c2c-product-name+} supports replication between sharded clusters. Individual shards are replicated in parallel from the source cluster to the destination cluster, however the source cluster's sharding configuration is not preserved.

Reply:
Done!